java/src/main/java/io/grpc/examples/wallet/AccountServer.java

```diff
@@ -97,17 +110,30 @@ private void start() throws IOException {
       Observability.registerExporters(gcpClientProject);
     }
     HealthStatusManager health = new HealthStatusManager();
+    ServerCredentials serverCredentials =
+        credentialsType == CredentialsType.XDS
+            ? XdsServerCredentials.create(InsecureServerCredentials.create())
+            : InsecureServerCredentials.create();
+    // Since the main server may be using TLS, we start a second server just for plaintext health
+    // checks
+    int healthPort = port + 1;
     server =
-        ServerBuilder.forPort(port)
+            XdsServerBuilder.forPort(port, serverCredentials)
```

Comment:
Just to confirm: XdsServerBuilder with insecure credentials will still work fine in the "regular" TD deployment where the servers are not xDS-managed, right? Want to make sure this will continue to work with the non-security user guide examples.

Reply:
Yes. This new code will continue to work as before for the old use cases (with the small addition of starting a health server on port + 1).